assets/js/components/Dashboard/NewPaMessagePage.tsx

```diff
@@ -129,15 +195,40 @@ const NewPaMessagePage = () => {
                 >
                   End
                 </Form.Label>
-                <div className="datetime-picker-group">
-                  <DatePicker
-                    selectedDate={endDate}
-                    onChange={setEndDate}
-                    minDateString={startDate}
-                    id="end-date-picker"
+                {associatedAlert.id && (
+                  <Form.Check
```

Comment:
You can actually use `Form.Switch` here and remove the `type`. Results in the same component being rendered.